lib/ood_core/job/adapters/kubernetes/templates/pod.yml.erb

```diff
@@ -148,6 +148,10 @@ spec:
   <%- end # if mount is [host,nfs] -%>
   <%- end # for each mount -%>
   <%- end # (configmap.to_s.empty? || all_mounts.empty?) -%>
+  <%- unless cluster.nil? -%>
+  nodeSelector:
+    cluster: <%= cluster %>
+  <%- end -%>
```

Comment:
Instead of implementing cluster specifically, seems like we should allow for `nodeSelector` configs. Then we at OSC can use `cluster`, but other folks will be able to use whatever they like.

Reply:
Ya, that would actually work out to halfway solve the problem of supporting GPUs since a GPU request is in resources but you can choose which kind of GPU you get via nodeSelector.  I'll close this and open new pull request to do node selector configuration